tests/conftest.py

```diff
@@ -20,15 +25,33 @@
 timeout = 120  # Timeout in seconds
 
 
-@contextmanager
+HTTP_LOGIN: str = "admin"
+HTTP_PASSWORD: str = "password"
+
+
+T4C_REPO_HOST: str = "127.0.0.1"
+T4C_REPO_NAME: str = "test-repo"
+T4C_PROJECT_NAME: str = "test-project"
+T4C_USERNAME: str = "admin"
+T4C_PASSWORD: str = "password"
+
+GIT_REPO_BRANCH: str = "backup-test"
+GIT_EMAIL: str = "exporter@example.com"
+GIT_USERNAME: str = "any"
+GIT_PASSWORD: str = "any"
+
+
+@contextlib.contextmanager
 def get_container(
     image: str,
     ports: dict[str, int | None] | None = None,
     environment: dict[str, str] | None = None,
     path: pathlib.Path | None = None,
     mount_path: str | None = None,
     network: str | None = None,
-) -> Iterator[containers.Container]:
+    entrypoint: list[str] | None = None,
+    use_docker_prefix: bool = True,
```

Comment:
Yes I could remove the `use_docker_prefix` here but I think it might be helpful in the future if we want to spin up an external image where we do not want to add our prefix and tag in front. Whats your opinion?

Reply:
If we need it in the future, we can add it in the future.

Reply:
> But then I would suggest renaming `image` to something like `image_suffix` so it is clear that you won't use exactly that image

I'd say that the suffix of an image is the tag. So `image_infix` would be better, but it is not really self-explaining. Better keep it as image and add a docstring if it's unclear.

Reply:
Isn't the format `image:tag` so hopefully no one would assume that `image_suffix` is the tag because then it would be named `tag`.